### PR TITLE
Abel - pass dialogue cat to ui manager

### PR DESCRIPTION
**Pull request by unoctium1**
Add cat to ui manager

### CODE REVIEW

File: Assets/_GameJamCat/Scripts/Managers/UIManager.cs [hunk 1]
```diff
@@ -8,30 +8,36 @@ namespace GameJamCat
...
-        
+
         [Title("HUD View")]
         [SerializeField] private TimerUI _timer = null;
         [SerializeField] private GameObject _crossHair = null;
         [SerializeField] private LivesViewBehaviour _livesView = null;
+
         [SerializeField] private GameObject _crossHairText = null;
-        
+        [SerializeField] private PlayerController _player = null; //I'm sorry for doing this I know its terrible
```
Comment (m4rtint) [inline Assets/_GameJamCat/Scripts/Managers/UIManager.cs]: you can just pass it in through worldmanager

Comment (unoctium1) [reply Assets/_GameJamCat/Scripts/Managers/UIManager.cs]: ah shit didn't realise world manager had the player, will do

Comment (unoctium1) [reply Assets/_GameJamCat/Scripts/Managers/UIManager.cs]: Fixed

**Comment by Gursaanj** [comment]
image

Is this supposed be null?

**Review by natibeltran777 — APPROVED** [review]

**Pull request merged**